docs/sdk/python/complementary-examples.md

```diff
@@ -29,18 +37,18 @@ set_network(Testnet)
 client = SolarClient('https://sxp.testnet.sh/api')
 
 # Step 1: Retrieve the incremental nonce of the sender wallet
-senderWallet = client.wallets.get('YOUR_SENDER_WALLET_ADDRESS')
+senderWallet = client.wallets.get('SENDER_WALLET_ADDRESS')
 nonce = int(senderWallet['data']['nonce']) + 1
 
 # Step 2: Create the transaction
-transaction = Transfer(
-    recipientId='YOUR_RECIPIENT_ADDRESS',
-    amount=200000000,
-    vendorField="Hello World"
-)
+transaction = Transfer()
+
 transaction.set_type_group(TRANSACTION_TYPE_GROUP.CORE)
 transaction.set_nonce(nonce)
-transaction.schnorr_sign('this is a top secret passphrase')
+transaction.add_payment(1, 'RECIPIENT_WALLET_ADDRESS_1')
+transaction.add_payment(2, 'RECIPIENT_WALLET_ADDRESS_2')
```

Comment:
`add_payment` => `add_transfer`

https://github.com/Solar-network/python-crypto/blob/c29a7188ffd58c7a1dc9178e294617ed0012f976/solar_crypto/transactions/builder/transfer.py#L31

Reply:
Thanks for review. Resolved in [ca0e45a](https://github.com/Solar-network/docs/pull/12/commits/ca0e45a01a79fd606008a7153e7e165598435120)